src/vs/editor/browser/widget/multiDiffEditorWidget/multiDiffEditorWidget.ts

```diff
@@ -45,6 +46,10 @@ export class MultiDiffEditorWidget extends Disposable {
 		this._register(recomputeInitiallyAndOnChange(this._widgetImpl));
 	}
 
+	public reveal(resource: IMultiDiffResource, range: IRange): void {
+		this._widgetImpl.get().reveal(resource, range);
```

Comment:
```suggestion
		this._widgetImpl.get().reveal(resource, Range.lift(range));
```
(not sure if the method name is really `lift` though)